addons/portal/models/mail_message.py

```diff
@@ -16,8 +16,10 @@ def portal_message_format(self):
 
     def _portal_message_format(self, fields_list):
         vals_list = self._message_format(fields_list)
+        note_id = self.env['ir.model.data'].xmlid_to_res_id('mail.mt_note')
```

Comment:
message_subtype_note_id ?

Side-note: not sure "xmlid_to_res_id" exists, isn't it "_xmlid_to_res_id" ?

Reply:
It seems that there are some occurences of `_xmlid_to_res_id` in the code but there is no method definition of it :smiley: